pinot-common/src/main/java/org/apache/pinot/common/audit/AuditConfig.java

```diff
@@ -36,7 +36,7 @@ public final class AuditConfig {
   private boolean _captureRequestPayload = false;
 
   @JsonProperty("capture.request.headers")
-  private boolean _captureRequestHeaders = false;
+  private String _captureRequestHeaders = "";
```

Comment:
nit: should this be renamed to `allowedRequestHeaders` instead since now it isn't a boolean but a string? (same for the JsonProperty and the setter / getter / log)

Reply:
Also, the getters/setters are already updated btw.